packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -323,17 +335,20 @@
                             </small>
                         </td>
 
-                        <!-- Country -->
+                        <!-- Campus -->
                         <td
                             class="text-center"
-                            data-ng-bind="ipBlock.country ? ipBlock.country.toUpperCase() : '-'"
+                            data-ng-bind="ipBlock.campus ? ipBlock.campus : '-'"
                         ></td>
 
                         <!-- Server -->
                         <td
                             data-ng-bind="ipBlock.routedTo.serviceName || '-'"
                         ></td>
 
+                        <!-- Virtual MAC -->
+                        <td></td>
```

Comment:
**question:** Why using an empty cell?

Reply:
@antleblanc the table has 2 different `tr`s and only one of them will be rendered based on the condition. this is a scenario where the value exists in the other `tr` and this particular `td` must be empty to maintain the consistency among the columns.